internal/configs/testdata/valid-files/null-version-module.tf

```diff
@@ -0,0 +1,9 @@
+variable "module_version" {
+  type    = string
+  default = null
+}
+
+module "foo" {
+  source  = "./foo"
+  version = var.module_version
```

Comment:
Instead of creating NullVersionModuleCallForTesting and passing that in, we could instead use a local instead of a variable in here.